common/autoware_overlay_rviz_plugin/autoware_mission_details_overlay_rviz_plugin/CMakeLists.txt

```
@@ -0,0 +1,132 @@
+cmake_minimum_required(VERSION 3.8)
+project(autoware_mission_details_overlay_rviz_plugin)
+
+# find dependencies
+find_package(ament_cmake_auto REQUIRED)
+find_package(ament_cmake REQUIRED)
+find_package(autoware_auto_vehicle_msgs REQUIRED)
+find_package(tier4_planning_msgs REQUIRED)
+find_package(autoware_perception_msgs REQUIRED)
+find_package(autoware_internal_msgs REQUIRED)
+ament_auto_find_build_dependencies()
+
+find_package(autoware_overlay_msgs REQUIRED)
+
+find_package(rviz_common REQUIRED)
+find_package(rviz_rendering REQUIRED)
+find_package(rviz_ogre_vendor REQUIRED)
+find_package(std_msgs REQUIRED)
```

You don't have to call find_packages() individually if you run ament_auto_find_build_dependencies()

```suggestion
ament_auto_find_build_dependencies()
```

---

This was done.

---

This wasn't applied @ahmeddesokyebrahim